source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRMediaRenderer.mm

```diff
@@ -81,16 +92,16 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     view.contentMode = UIViewContentModeScaleAspectFill;
     contentholdingview.isMediaType = YES;
 
+
     // process play icon image
-    NSString *piikey = [NSString stringWithCString:[acoConfig getHostConfig] -> GetMedia().playButton.c_str() encoding:[NSString defaultCStringEncoding]];
-    UIImage *playIconImage = imageViewMap[piikey];
+    NSString *playIconKey = makeKeyForImage(acoConfig, @"media-playicon-image", pieces);
```

Comment:
>media-playicon-image [](start = 57, length = 20)

Avoid hard coded literals in the code - where is this coming from? Also is this going to have potential localization impact? SHould this be out in its own const strings class for easy localizability potentially?

Reply:
there will be no localization impact. the strings are only used internally.

Reply:
In general bad practice to hard code string literals - whether they are to be localized or not.

---
In reply to: [509830950](https://github.com/microsoft/AdaptiveCards/pull/4897#discussion_r509830950) [](ancestors = 509830950)